Make processed part of script read only when Coq is busy (#15714)
coq/coq: doc/changelog/09-coqide/15714-lost_keystrokes.rst

- **Fixed:**
  Attempted edits to the processed part of a buffer while

>   Coq is processing an XML request are now ignored to

Reviewer: ```suggestion
  Coq is busy evaluating a command are now ignored to
```
It is not about the wording, it is just that I don't expect users to even know that the protocol is XML based (and the changelog is for them)

Author: Good point.  Made it "Coq is busy processing a request are now ignored to"

Can also be tactics as well as things like "setoption".

Reviewer 2: Coq or CoqIDE though? Perhaps it doesn't matter since it's in the CoqIDE section.

Author: I think "Coq" is most accurate.  CoqIDE is the obvious intermediary to talk to Coq.
